js/everything.js

```diff
@@ -580,10 +584,11 @@ $(function() {
     $("#form").submit(function(event) {
         event.preventDefault();
         var formValue = $.trim($("#v").val());
-        var formValueTime = /[?&]t=(\d*)/g.exec(formValue);
-        if (formValueTime && formValueTime.length > 1) {
-            formValueTime = parseInt(formValueTime[1], 10);
-            formValue = formValue.replace(/&t=\d*$/g, "");
+        var formValueTime = /[?&](t|time_continue)=(\d+)/g.exec(formValue);
+        if (formValueTime && formValueTime.length > 2) {
+            formValueTime = parseInt(formValueTime[2], 10);
+            formValue = formValue.replace(/[?&]t=\d*/g, "");
```

Comment:
We can be a bit more selective here, and use `formValueTime[1]` (which will be either `t` or `time_continue`) so we don't have to do the replacement twice

Reply:
OK, I thought the input could possibly contain both. If that's not the case, I'll change it.